src/Autohide.js

```diff
@@ -182,10 +186,55 @@ define(function (require, exports, module) {
         exposeDelay = delay;
     }
 
+    function showDelayDialog() {
+        var Dialog = Dialogs.showModalDialog(
+            brackets.DIALOG_ID_SAVE_CLOSE,
+            Strings.AUTOHIDE_DELAY_MODAL_TITLE,
+            Mustache.render(sliderTemplate, {
+                exposeDelay: exposeDelay,
+                millisecondsLabel: Strings.AUTOHIDE_DELAY_MODAL_MS_LABEL
+            })
+        );
+
+        var $slider = $("#outline-delay-slider");
+        var $inputbox = $("#outline-delay-inputbox");
+        var inputboxCurrentValue = $inputbox.val();
+
+        Dialog.done(function (buttonId) {
+            if (buttonId === "ok") {
+                prefs.set("autohideDelay", $slider.val());
```

Comment:
This sets the preference as string. While that still works, I would prefer it to be set as int.

Reply:
Fixed.